test.properties

```diff
@@ -8366,7 +8366,7 @@
     #
 
     search.engine[search-remote-minimum]=remote-elasticsearch
-    suite.elastic.version[search-remote-minimum]=7.16.3
+    suite.elastic.version[search-remote-minimum]=7.17.0
```

Comment:
@xbrianlee Are we sure about this bump?
on https://liferay.atlassian.net/browse/LPS-195831 under Related-tasks (QA / Engineering)
see point 3: ci:search-remote-minimum should continue be using 7.16.3 to match the client's version

Reply:
ya it looks good, i think the rule is, if you bump the client, you have to bump the server too.

Reply:
at least for "guaranteed" compatibility

Reply:
although i thought thats why we have the minimum suite, to test if things still work with the "unguaranteed" situation? @lipusz

Reply:
We can bump the minimum suite also to 7.17. Elasticsearch does not "support" connecting a newer client to an older minor version server anyways.